go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/giantswarm/certs/v3 v3.1.0
 	github.com/giantswarm/ipam v0.2.0
 	github.com/giantswarm/k8sclient/v5 v5.0.0
-	github.com/giantswarm/k8scloudconfig/v10 v10.0.0
+	github.com/giantswarm/k8scloudconfig/v9 v9.3.0
```

Comment:
if this change will be merged into master we need to have k8scc v10

Reply:
if not then please change the branch you re-merging into to avoid confusion :p

Reply:
this was only because aws-operator does not work in master with 1.18, I'll revert it after finishing tests.

Reply:
ACK, in case you want to test it on 1.19 you could use the 14.0.0 release https://github.com/giantswarm/releases/pull/586
 
 ```
 opsctl release deploy -i ginger -b 14.0.0 -r releases@aws-1-19 -c aws-operator@allow-nfs-traffic
 ```